lib/features/search-pad/SearchPad.js

```diff
@@ -525,7 +486,7 @@ function createHtmlText(tokens) {
 
   tokens.forEach(function(t) {
     if (t.matched) {
-      htmlText += '<strong class="' + SearchPad.RESULT_HIGHLIGHT_CLASS + '">' + escapeHTML(t.matched) + '</strong>';
+      htmlText += '<b>' + escapeHTML(t.matched) + '</b>';
```

Comment:
Why do we remove the ability to customize here?

Reply:
Also, let's keep using semantic html. `<strong>` for important stuff, `<em>` for emphasis.

Reply:
In terms of a11y, `<strong>` matters to screen readers while `<b>` doesn't which is what I'd expect in that case. I wouldn't expect the screen reader to focus on specific parts of the results.

Reply:
:+1: for "just for decoration.

Reply:
I like how the tag meaning was changed from "<b>oldface" to "<b>ring attention". I'm ok with leaving it as is.